bundle-restricted/manifests/ibm-namespace-scope-operator-restricted.clusterserviceversion.yaml

```
@@ -47,6 +47,33 @@ spec:
     mediatype: image/png
   install:
     spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - roles
+          - rolebindings
+          verbs:
+          - get
+          - list
+        - apiGroups:
+          - ''
+          resources:
+          - configmaps
+          - pods
+          verbs:
+          - get
+          - list
+        - verbs:
+          - get
+          - list
+          - watch
+          apiGroups:
+          - operator.ibm.com
+          resources:
+          - namespacescopes
```

```suggestion
        - apiGroups:
          - operator.ibm.com
          resources:
          - namespacescopes
          verbs:
          - get
          - list
          - watch
```
Should follow the format of above blocks

---

also, shouldn't permissions be changed in the `rbac` folder and then regenerate the bundle to get a new CSV, instead of changing the CSV file directly?

---

NamespaceScope operator is a special one. There are two bundles with the same code base, so it's not convenient for us to use operator-sdk to generate CSV.